qemu/tests/kvm_stat.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+import subprocess
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1.install kernel-tools package in host
+    2.modify kvm_stat.service file and start kvm_stat.service
+    3.run VM
+    4.use "logrotate" to separate the log file
+    5.Check the generated log file
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    interval = float(params.get("interval_time", "10"))
+
+    def tweak_file(filename, old_str, new_str):
+        # replace the strings in the file
+        file_data = ""
+        with open(filename, "r") as f:
+            for line in f:
+                if old_str in line:
+                    line = line.replace(old_str, new_str, 1)
+                file_data += line
+        with open(filename, "w") as f:
+            f.write(file_data)
+
+    def separate_file(file_path):
+        num = 1
+        file_size = os.path.getsize(file_path)
+        if num < 6 and file_size > 10240:
+            process.system('logrotate -v /etc/logrotate.d/kvm_stat')
+            num += 1
+        else:
+            return False
+
+    def check_log():
+        check_file = params.get("check_file")
+        check_file_cmd = subprocess.Popen(check_file, shell=True, stdout=subprocess.PIPE)
+        out = check_file_cmd.stdout.readlines()
+        for line in out:
+            result = line.strip().decode()
+            if 'kvm_stat.csv.5.gz' in result:
+                logging.info("Test complete")
+
+    logging.info("Install kernel-tools package in host")
+    pkg_name = params.get("pkg_name")
+    utils_package.package_install(pkg_name)
+    if not utils_package.package_install("kernel-tools"):
+        test.cancel("Install dependency packages failed")
```

Comment:
Update L57~L59:
```
pkgs = params.objects("pkg_name")
if not utils_package.package_install(pkgs):
    test.cancel(****)
```

Reply:
HI Yanan,
I've updated it, Please review this patch again.thanks a lot.